src/App.jsx

```diff
@@ -1,13 +1,96 @@
-
-import './App.css'
+import './App.css';
+import './css-reset.css';
+import FruitCounter from './components/fruit-counter/fruit-counter.jsx';
+import React, { useState } from 'react';
+import OrderForm from './components/order-form/order-form.jsx';
 
 function App() {
+  const initialProductState = [
```

Comment:
Slim gebruik gemaakt van een initiële variabele! Hierdoor is de reset functie heel makkelijk te implementeren.